src/App.css

```diff
@@ -0,0 +1,50 @@
+body {
+  margin: 0;
+  background: linear-gradient(to right, #8484c2, aqua);
+}
+
+.calContainer {
+  place-items: center;
+}
+
+.output {
+  grid-column: 1/ -1;
+  background-color: rgba(0, 0, 0, 0.75);
+  display: inline;
+}
+
+.output .previous-operand {
+  color: rgba(255, 255, 255, 0.75);
+  font-size: 1.5rem;
+}
+
+.output .current-operand {
+  color: white;
+  font-size: 2.5rem;
+}
+
+.calculator-grid {
+  display: grid;
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+  justify-content: center;
+  grid-template-columns: repeat(4, 6rem);
+  grid-template-rows: minmax(7rem, auto) repeat(5, 6rem);
+}
+
+.calculator-grid button {
+  cursor: pointer;
+  font-size: 2rem;
+  border: 1px solid white;
+  outline: none;
+  background-color: rgba(255, 255, 255, 0.75);
+}
+
+.calculator-grid button:hover,
+.calculator-grid button:focus {
+  background-color: rgba(255, 255, 255, 0.9);
+}
+
+.span-two {
+  grid-column: span 2;
+}
```

Comment:
- [ ] [REQUIRED]  Good job so far! The `Calculator.js` component should match the following given design:

![image](https://user-images.githubusercontent.com/63915024/191551513-d6156eb2-8a7f-43b8-8dcd-b0a7197e3517.png)